lxd/network/driver_ovn.go

```diff
@@ -4717,8 +4717,10 @@ func (n *ovn) ForwardUpdate(listenAddress string, req api.NetworkForwardPut, cli
 		}
 
 		newForward := api.NetworkForward{
-			ListenAddress:     curForward.ListenAddress,
-			NetworkForwardPut: req,
+			ListenAddress: curForward.ListenAddress,
```

Comment:
As req is a api.NetworkForwardPut we should be using api.NetworkForward.SetWritable() here to avoid duplication.